common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivity.java

```diff
@@ -75,6 +102,8 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
                 .setTransitionStyle(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
                 .replace(R.id.authorization_activity_content, mFragment)
                 .commit();
+
+        adjustLayout();
```

Comment:
This seems to early to detect rotation. What if the `Activity` is created, user leaves, rotates device, and returns?

Do we need to move this to `onResume()`?

Reply:
Oh, I see there's also a call in `onConfigurationChanged()`

I guess this is fine?